content/c-sharp/concepts/strings/terms/format/format.md

```diff
@@ -0,0 +1,237 @@
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
+| `0`       | Digit placeholder replaced by the digit of the formatted value at the same position, or by a `0` if there is no digit.                                              | `00000` >> `01235`      |
+| `#`       | Digit placeholder replaced by the digit of the formatted value at the same position, or by nothing if there is no digit.                                            | `#` >> `12345`          |
+| `.`       | Decimal separator, based on the culture. Digit placeholders on the right decide how to format the decimal part.                                                     | `0.00` >> `1234.50`     |
+| `,`       | A comma between two digit placeholders in the integral part indicates to split the integral digits into groups. The actual group separator is based on the culture. | `#,#.#` >> `1,234.5`    |
+| `;`       | A semicolon separates the different sections of the format for positive, zero, and negative values ​​(in that order). The zero section can be skipped.              | `-(-#);#` >> `-(-1235)` |
+| `\`       | Escape character to display characters otherwise interpreted as specifiers. It must be doubled in quoted string literals.                                           | `\##` >> `#1235`        |
+| any other | Displayed as is.                                                                                                                                                    | `≈#` >> `≈1235`         |
+
+### Date Formats
+
+The following table shows some of the commonly used standard formats for date values and how they would format a date and time value corresponding to June 7, 2009, at 8:05:09 PM, with the US culture.
+
+| Specifier | Description                  | Example                           |
+| --------- | ---------------------------- | --------------------------------- |
+| `d`       | Short date.                  | `6/7/09`                          |
+| `D`       | Long date.                   | `Sunday, June 7, 2009`            |
+| `f`       | Full date and short time.    | `Sunday, June 7, 2009 8:05 PM`    |
+| `F`       | Full date and long time.     | `Sunday, June 7, 2009 8:05:09 PM` |
+| `g`       | General date and short time. | `6/7/09 8:05 PM`                  |
+| `G`       | General date and long time.  | `6/7/09 8:05:09 PM`               |
+| `M`       | Month and day.               | `June 7`                          |
+| `t`       | Short time.                  | `8:05 PM`                         |
+| `T`       | Long time.                   | `8:05:09 PM`                      |
+| `Y`       | Month and year.              | `June 2009`                       |
```

Comment:
The description here is not a full sentence, so we can skip using a full stop here.

```suggestion
| Specifier | Description                  | Example                           |
| --------- | ---------------------------- | --------------------------------- |
| `d`       | Short date                  | `6/7/09`                          |
| `D`       | Long date                   | `Sunday, June 7, 2009`            |
| `f`       | Full date and short time    | `Sunday, June 7, 2009 8:05 PM`    |
| `F`       | Full date and long time     | `Sunday, June 7, 2009 8:05:09 PM` |
| `g`       | General date and short time | `6/7/09 8:05 PM`                  |
| `G`       | General date and long time  | `6/7/09 8:05:09 PM`               |
| `M`       | Month and day               | `June 7`                          |
| `t`       | Short time                  | `8:05 PM`                         |
| `T`       | Long time                   | `8:05:09 PM`                      |
| `Y`       | Month and year              | `June 2009`                       |
```